File: app/src/main/java/com/lazylee/apiguidedemo/coretopics/ui_navigation/canvas/CanvasThreeActivity.java
package com.lazylee.apiguidedemo.coretopics.ui_navigation.canvas;

import androidx.appcompat.app.AppCompatActivity;
import android.os.Bundle;

import com.lazylee.apiguidedemo.coretopics.ui_navigation.custom.ClippedView;

public class CanvasThreeActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(new ClippedView(this));
    }
}

File: app/src/main/java/com/lazylee/apiguidedemo/coretopics/ui_navigation/canvas/CanvasThreeActivity.kt
package com.lazylee.apiguidedemo.coretopics.ui_navigation.canvas

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import com.lazylee.apiguidedemo.coretopics.ui_navigation.custom.ClippedView

class CanvasThreeActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(ClippedView(this))
    }
}